soh/CMakeLists.txt

```diff
@@ -579,6 +579,7 @@ if (CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang|AppleClang")
 
         target_compile_options(${PROJECT_NAME} PRIVATE
             -Wall -Wextra -Wno-error
+            -Wno-format-security
```

Comment:
I'd like to address the format warnings with imgui

Reply:
@serprex done